src/common/components/clusterConfiguration/DiscoveryImageConfigForm.tsx

```diff
@@ -100,6 +100,7 @@ export const DiscoveryImageConfigForm: React.FC<DiscoveryImageConfigFormProps> =
       onSubmit={handleSubmit}
     >
       {({ submitForm, isSubmitting, status }) => {
+        const { error } = status as unknown as StatusErrorType;
```

Comment:
Looks like `status` can be null?

Reply:
Yes, status can be null or be undefined

Reply:
Then the new code could fail. The old code did `status?.error`, but now `error` is extracted unconditionally